checks.d/redisdb.py

```diff
@@ -215,6 +218,17 @@ def _check_db(self, instance, custom_tags=None):
                             self.warning("{0} key not found in redis".format(key))
                         self.gauge('redis.key.length', 0, tags=key_tags)
 
+        # Save the replication delay for each slave
+        for key in info:
+            if self.slave_key_pattern.match(key) and isinstance(info[key], dict):
+                slave_offset = info[key].get('offset')
+                master_offset = info.get('master_repl_offset')
+                if slave_offset and master_offset and master_offset - slave_offset >= 0:
+                    delay = master_offset - slave_offset
+                    slave_tags = tags + ['slave_ip:%s' % info[key]['ip']] if 'ip' in info[key] else tags
```

Comment:
It might be more readable this way:
```python
slave_tags = tags
for slave_tag in ('ip', 'port'):
    if slave_tag in info[key]:
        slave_tags.append('slave_{0}:{1}'.format(slave_tag, info[key][slave_tag]))
```